Cargo.toml

```diff
@@ -132,7 +132,7 @@ style_malloc_size_of = { package = "malloc_size_of", git = "https://github.com/s
 surfman = { git = "https://github.com/servo/surfman", rev = "300789ddbda45c89e9165c31118bf1c4c07f89f6", features = ["chains"] }
 syn = { version = "2", default-features = false, features = ["clone-impls", "derive", "parsing"] }
 synstructure = "0.13"
-taffy = { version = "0.7", default-features = false, features = ["std", "serde", "grid"] }
+taffy = { git = "https://github.com/dioxuslabs/taffy", rev = "e353c141", default-features = false, features = ["std", "serde", "grid"] }
```

Comment:
Why not release a new taffy version?

Reply:
There's a bit of API cleanup required before we'll be ready to make a release for public consumption. We can wait to land this until that happens if you think that's better?

(personally I'm a bit torn: On the one hand I think that Servo ought to treat Taffy as a pseudo-first-party library similar to other libraries which it takes a git dependency on. On the other hand I think Servo really ought to move towards depending on crates.io releases for those libraries too!)

I also wasn't sure if further Taffy changes would be required in response to review feedback on this PR.

Reply:
Update: the API cleanup has done so I could do a new release. However, I'd like to include the fixes from https://github.com/servo/servo/pull/34948 in a new taffy release. So basing this PR on such a release would mean mixing in those changes into this PR. I figure it's easier to review if there's one feature per PR, but I can combine them if that's preferable.